### PR TITLE
Doc upgrade paley graph

### PR DESCRIPTION
**Pull request by paulitapb**
I added latex formatting to the doc to properly express some math expressions.

### CODE REVIEW

**Comment by dschult** [comment]
Looks like the latex commands within dollar signs need each slash doubled to indicate a latex slash.

**Comment by rossbar** [comment]
> Looks like the latex commands within dollar signs need each slash doubled to indicate a latex slash.

Making the docstring a raw string (e.g. `r"""`) may be another option.

There definitely is some weirdness in the formatting, but the motivation for the change is good! Here's the rendered page (as of d3bff2c) for reference: https://output.circle-artifacts.com/output/job/e1d60a12-2f31-4761-96a1-71d53775f18e/artifacts/0/doc/build/html/reference/generated/networkx.generators.expanders.paley_graph.html#paley-graph

**Comment by paulitapb** [comment]
> Looks like the latex commands within dollar signs need each slash doubled to indicate a latex slash.
ahh, the pre-commit was doing exactly that but I changed it back because I thought it was wrong.  I will run the pre-commit and push again.

File: networkx/generators/expanders.py [hunk 1]
```diff
@@ -146,19 +146,19 @@ def chordal_cycle_graph(p, create_using=None):
 
 
 def paley_graph(p, create_using=None):
-    """Returns the Paley $\frac{(p-1)}{2}-$regular graph on $p$ nodes.
+    """Returns the Paley $\\frac{(p-1)}{2}-$regular graph on $p$ nodes.
```
Comment (dschult) [inline networkx/generators/expanders.py:149]:
```suggestion
    """Returns the Paley $\\frac{(p-1)}{2}$ -regular graph on $p$ nodes.
```

File: networkx/generators/expanders.py [hunk 2]
```diff
@@ -146,19 +146,19 @@ def chordal_cycle_graph(p, create_using=None):
...
 
-    The returned graph is a graph on $ \mathbb{Z}/p\mathbb{Z}$ with edges between $x$ and $y$
-    if and only if $x-y$ is a nonzero square in $\mathbb{Z}/p\mathbb{Z}$.
+    The returned graph is a graph on $ \\mathbb{Z}/p\\mathbb{Z}$ with edges between $x$ and $y$
```
Comment (dschult) [inline networkx/generators/expanders.py:151]:
```suggestion
    The returned graph is a graph on $\\mathbb{Z}/p\\mathbb{Z}$ with edges between $x$ and $y$
```

**Comment by paulitapb** [comment]
> The formatting looks correct in the rendered docs now.
> 
> One thing to note about these proposed changes is that, while they make the html more readable due to the LaTeX rendering, they arguably make the docstring itself _less_ readable since raw LaTeX is generally harder to read in the terminal than inline-literal math.
> 
> I'm mostly bringing this up here to point out that we probably don't want to go making similar changes in all the docstrings without having a wider discussion about what the policy should be. Since it's already done in this case (and things are still pretty readable) I'm +1 for putting it in. Thanks @paulitapb

That's a good point that I haven't considered before. We definitely don't want our docstrings to look like raw Latex but it also makes the HTML version more readable and nice. It's a good topic for discussion.

**Review by dschult — APPROVED** [review]
Sounds/looks good to me.

**Pull request merged**